toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1001 @@
+    </message>
+    <message>
+        <source>Preset:</source>
+        <translation>Vorlage:</translation>
+    </message>
+    <message>
+        <source>Break Sharp Angles</source>
+        <translation type="vanished">Scharfe Winkel brechen</translation>
+    </message>
+    <message>
+        <source>Pencil Mode</source>
+        <translation type="vanished">Bleistift Modus</translation>
+    </message>
+    <message>
+        <source>Pressure Sensitivity</source>
+        <translation type="vanished">Druckempfindlichkeit</translation>
+    </message>
+    <message>
+        <source>Cap</source>
+        <translation>Cap</translation>
+    </message>
+    <message>
+        <source>Join</source>
+        <translation>Vereinen</translation>
+    </message>
+    <message>
+        <source>Miter:</source>
+        <translation>Gehrung:</translation>
+    </message>
+    <message>
+        <source>Size</source>
+        <translation>Größe</translation>
+    </message>
+    <message>
+        <source>Break</source>
+        <translation>Scharfe Winkel brechen</translation>
+    </message>
+    <message>
+        <source>Pencil</source>
+        <translation>Bleistift Modus</translation>
+    </message>
+    <message>
+        <source>Pressure</source>
+        <translation>Druck</translation>
+    </message>
+</context>
+<context>
+    <name>BrushToolOptionsBox</name>
+    <message>
+        <source>Preset Name</source>
+        <translation>Name der Vorlage</translation>
+    </message>
+    <message>
+        <source>OK</source>
+        <translation>OK</translation>
+    </message>
+    <message>
+        <source>Cancel</source>
+        <translation>Abbruch</translation>
+    </message>
+</context>
+<context>
+    <name>ControlPointEditorTool</name>
+    <message>
+        <source>Auto Select Drawing</source>
+        <translation>Zeichnung automatisch wählen</translation>
+    </message>
+</context>
+<context>
+    <name>ControlPointSelection</name>
+    <message>
+        <source>Set Linear Control Point</source>
+        <translation>Linearen Kontrollpunkt setzen</translation>
+    </message>
+    <message>
+        <source>Set Nonlinear Control Point</source>
+        <translation>Nicht-Linearen Kontrollpunkt setzen</translation>
+    </message>
+</context>
+<context>
+    <name>EditTool</name>
+    <message>
+        <source>Scale Constraint:</source>
+        <translation>Maßstab-Beschränkung</translation>
+    </message>
+    <message>
+        <source>Auto Select Column</source>
+        <translation>Spalte automatisch wählen</translation>
+    </message>
+    <message>
+        <source>Global Key</source>
+        <translation>Globale Taste</translation>
+    </message>
+    <message>
+        <source>Lock Center E/W</source>
+        <translation>Mitte [X] sperren</translation>
+    </message>
+    <message>
+        <source>Lock Center N/S</source>
+        <translation>Mitte［Y］ sperren</translation>
```

Comment:
On second observation, the Japanese text just says Bezugspunkt [X] (Point of Reference [X]) here.